app/models/bank_account.rb

```diff
@@ -19,4 +19,16 @@ def holders
   def display_name
     "#{bank_provider.name} Acct #{account_number}"
   end
+
+  def receives_tax_credits?
+    true # TODO: CCMS placeholder
+  end
+
+  def receives_wages?
+    true # TODO: CCMS placeholder
+  end
+
+  def receives_benefits?
+    true # TODO: CCMS placeholder
+  end
```

Comment:
Is `BankAccount` really the right place for these fields. I would expect to see them in the `Applicant` model. `has_one :applicant, through: :legal_aid_application` could be added to `BankAccount` and the methods delegated to make then easily accessible from instances of this model.

If these methods are flags pertaining to the nature of the data stored within a particular set of bank transactions (for which `BankAccount` is the container) would `includes_` or `contains_` be a better prefix. For example `includes_tax_credits?`

Reply:
OK - but "receiving benefit" has a specific meaning and is something that applies to a person and not a bank account. Surely `includes_benefits?` or `contains_benefits?` is clearer as to it's meaning.